scenarios/bigquery.py

```diff
@@ -40,56 +40,56 @@ def check(*cmd, **keyargs):
 def validate_metric_name(name):
     """Validates that a string is useable as a metric name (Can be used as GCS object name)."""
     if len(name) == 0:
-        raise ValueError("metric-name cannot be an empty string.")
-    if any(char in name for char in "\r\n[]*?#/\\"):
-        raise ValueError("metric-name cannot contain any line feeds, carriage returns, or the "
-                         "following characters: []*?#/\\")
+        raise ValueError('metric-name cannot be an empty string.')
+    if any(char in name for char in '\r\n[]*?#/\\'):
```

Comment:
How about `re.match(r'[\w-]+')`, I would emphasize simplicity over flexibility. Do we really need to allow a metric named `!&*)`?